drop/modules/aberrant-splicing-pipeline/FRASER/04_fit_hyperparameters_FraseR.R

```diff
@@ -20,6 +20,9 @@
 saveRDS(snakemake, snakemake@log$snakemake)
 source(snakemake@params$setup, echo=FALSE)
 
+if ("random_seed" %in% names(snakemake@config) && isTRUE(snakemake@config$random_seed))
+  set.seed(42)
```

Comment:
Also allow for numeric input. So you can set the seed. Should/could be done for all files where randomness is happening. 

```suggestion
if ("random_seed" %in% names(snakemake@config)){
  rseed <- snakemake@config$random_seed
  if(TRUE(rseed)){
    set.seed(42)
  } else if (is.numeric(rseed)){
    set.seed(as.integer(rseed))
  }
}
```

Reply:
why TRUE(rseed) and not isTrue(rseed)?

Reply:
thanks!

Reply:
will `rseed <- snakemake@config$random_seed` be read as character or logical if it contains `true`?